documentation/assemblies/configuring/assembly-reassign-tool.adoc

```diff
@@ -16,10 +16,16 @@ The tool can also be used to reassign partitions and balance the distribution of
 However, it is recommended to use xref:cruise-control-concepts-str[Cruise Control for automated partition reassignments and cluster rebalancing]. 
 Cruise Control can move topics from one broker to another without any downtime, and it is the most efficient way to reassign partitions.
 
+NOTE: It is recommended to run the `kafka-reassign-partitions.sh` tool as a separate interactive pod rather than within the broker container.
```

Comment:
should we be clear about what "interactive" pod mean? So explicitly mentioning the `-i` parameter on the `kubectl run` command?

Reply:
I've changed to:

> NOTE: It is recommended to run the `kafka-reassign-partitions.sh` tool as a separate interactive pod rather than within the broker container.
> Running the Kafka `bin/` scripts within the broker container may cause a JVM to start with the same settings as the Kafka broker, which can potentially cause disruptions.
> By running the `kafka-reassign-partitions.sh` tool in a separate pod, you can avoid this issue.
> Running a pod with the `-ti` option creates an interactive pod with a terminal for running shell commands inside the pod.